### PR TITLE
add volcano camera config tools

### PR DESCRIPTION
**Pull request by ozym**
This PR adds two config tools for camera images.

The first is used simply to caption an image if needed, it uses the mounts and views tables.

The second is related to volcams, so adds in some extra detail regarding actual volcano targets.

### CODE REVIEW

File: cmd/volcam-config/main.go [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+	flag.Parse()
+
+	set, err := delta.NewBase(base)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	var volcs []Volcam
+	for _, l := range set.Views() {
```
Comment (CallumNZ) [inline cmd/volcam-config/main.go:68]: An alternative naming suggestion:
`for _, v := range set.Views()`
`for _, m := range set.Mounts()`
`for _, c := range set.InstalledCameras()`
`for k, vals := range keywords {`
`for _, val := range vals {`
`for k, val := range targets {`

**Review by CallumNZ — COMMENTED** [review]
Looks good, just a naming suggestion.

File: cmd/volcam-config/main.go (hunk 1 above)
Comment (ozym) [reply cmd/volcam-config/main.go]: Good suggestions, I've aimed to make the variables a little more readable. Have gone for longer form where it's a struct being returned and shorter for looping like variables.

**Comment by ozym** [comment]
@CallumNZ have updated variable names etc.

**Review by CallumNZ — APPROVED** [review]

**Pull request merged**